tests/integration/release/test_runner.py

```diff
@@ -19,7 +19,7 @@
 )
 def test_release_tests(pytest_args: list[str]) -> None:
     os.chdir(Path(__file__).parent)  # need this to access cli options from conftest.py
-    base_cmd = ["pytest", "--release", "test_release.py"]
+    base_cmd = ["pytest", "--release", "test_release.py", "--check-max-tb=1000"]
```

Comment:
what is this doing?

Reply:
This flag controls how many failures are shown at once and, oddly enough, defaults to 1.